src/ui/components/SelectRegistrar.tsx

```diff
@@ -79,7 +79,7 @@ type RegistrarItemProps = {
 };
 
 function RegistrarItem({onSelect, registrar}: RegistrarItemProps) {
```

Comment:
I've just realized we are doing the same in multiple places, I'll create an issue to refactor a bit all these components. We already have a `Select` component, we just need to make it a bit more generic to reuse it in all these cases